### PR TITLE
70 buttonfix

### PR DESCRIPTION
**Pull request by ericx057**
Final code ready for review.

Ignore 102 fix, not sure why pushed to this pull req

### CODE REVIEW

**Comment by Shawk031** [comment]
test comment

File: app/components/elements/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+interface ButtonProps {
+  className?: string;
+  children?: React.ReactNode;
+  name?: string;
+  value?: string;
+  type?: 'button' | 'submit' | 'reset';
+}
```
Comment (hangler) [inline app/components/elements/Button.tsx:7]: Let's have these `ButtonProps` extend the actual props for a `<button>` element. An example of how to do this with TypeScript is here: https://stackoverflow.com/a/62568833/715870

Then, you can delete the members of this `ButtonProps` interface that simply repeat existing props — in this case, that appears to be every member. However, what we can then do to make this really useful is add something like:

```tsx
interface ButtonProps {
  size?: 'small' | 'medium' | 'large' // We'll want to set 'medium' as the default later if this isn't passed
  appearance?: 'primary' | 'secondary' | 'danger' | 'success' // Set different colours based on this prop
}
```

Hopefully this makes sense. Let me know if not!

**Review by hangler — CHANGES_REQUESTED** [review]
Thanks for this! My main feedback would be to incorporate some changes to make it easier for developers to use your component, rather than simply re-wrapping a `<button>` element.

**Review by hangler — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**